packages/location-field/src/styled.tsx

```diff
@@ -130,7 +127,11 @@ export const MenuItemA = styled.a`
   white-space: nowrap;
 `;
 
-export const MenuItemHeader = styled.li`
+export const MenuItemHeader = styled.li<{
+  fgColor?: string;
+  bgColor?: string;
+  centeredText?: boolean;
```

Comment:
Sort props

Reply:
Addressed in 18d5606109e7dfce7d00d42b42c8b1b6f075a0bc